modules/nf-core/ensemblvep/main.nf

```diff
@@ -2,13 +2,13 @@ process ENSEMBLVEP {
     tag "$meta.id"
     label 'process_medium'
 
-    conda (params.enable_conda ? "bioconda::ensembl-vep=106.1" : null)
+    conda (params.enable_conda ? "bioconda::ensembl-vep=108.2" : null)
     container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
-        'https://depot.galaxyproject.org/singularity/ensembl-vep:106.1--pl5321h4a94de4_0' :
-        'quay.io/biocontainers/ensembl-vep:106.1--pl5321h4a94de4_0' }"
+        'https://depot.galaxyproject.org/singularity/ensembl-vep:108.2--pl5321h4a94de4_0' :
+        'quay.io/biocontainers/ensembl-vep:108.2--pl5321h4a94de4_0' }"
 
     input:
-    tuple val(meta), path(vcf)
+    tuple val(meta), path(vcf, stageAs: "?/*")
```

Comment:
why the stageAs?

Reply:
you don't seem too sure about it

Reply:
Yep, to avoid any possible name collisions. FWIW, I think `stageAs` should be added to almost all file inputs

Reply:
I agree no harm in that!